src/SassBuilder.php

```diff
@@ -97,8 +128,76 @@ public static function guessCssNameFromSassFile(string $sassFile, string $output
         return $outputDirectory.'/'.$fileName.'.output.css';
     }
 
-    private function createBinary(): SassBinary
+    /**
+     * @return list<string>
+     */
+    public function getBuildOptions(): array
     {
-        return new SassBinary($this->projectRootDir.'/var', $this->binaryPath, $this->output);
+        $buildOptions = [];
+        $options = [...self::SASS_OPTIONS, ...$this->sassOptions];
+        foreach ($options as $option => $value) {
+            // Set only the defined options.
+            if (null === $value) {
+                continue;
+            }
+            // --no-embed-source-map
+            // --quiet
+            if (str_starts_with($option, '--[no-]')) {
+                $buildOptions[] = str_replace('[no-]', $value ? '' : 'no-', $option);
+                continue;
+            }
+            // --style=compressed
+            if (\is_string($value)) {
```

Comment:
And call it like this from the build command: 

```
  $args = [
      ...$this->getScssCssTargets(),
      ...$this->getBuildOptions(['--watch' => $watch]),
  ];
```

That will also allow us to change the signature (later) from `build($watch)` to `build(arra $options) ` and so we could expose **some** options in the BuildCommand 

WDYT ?

Reply:
I pushed this idea, i can revert anytime obviously :)

Reply:
Wait, not sure it will work... as `--watch` value IS a string, i.e. we will fall into that `if (\is_string($value)) {` and do something like `--watch=--watch` what we don't need, right? it seems the previous solution was better

Reply:
https://github.com/SymfonyCasts/sass-bundle/pull/39/files

It's now 

```php
...$this->getBuildOptions(['--watch' => $watch]),
```

So that should work as before, but in a more "generic" way